src/sentry/migrations/0909_emerge_upload_models.py

```diff
@@ -0,0 +1,165 @@
+# Generated by Django 5.2.1 on 2025-05-21 22:48
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+import sentry.db.models.fields.jsonfield
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0908_increase_email_field_length"),
```

Comment:
@wedamija Should they put their migrations in their prepod folder?  Starting at 0001_*?  Replay did this.  I'm not sure if that's a standard approach or if we want them in the global migrations folder.

Reply:
Yeah, definitely store the migrations in this folder and make it it's own app

Reply:
Awesome that's great, will try it out.